lizmap/modules/lizmap/lib/Project/configFile.php

```diff
@@ -0,0 +1,389 @@
+<?php
+
+namespace Lizmap\Project;
+
+class configFile
+{
+    protected $data;
+
+    public function __construct($cfgFile)
+    {
+        $this->data = json_decode($cfgFile);
+        if (!$this->data) {
+            return null;
```

Comment:
a constructor never returns a value, right?

IMHO, you should throw an exception instead, if json_decode has failed (the returned value is not an object).